tests/angular_devkit/build_webpack/angular-app/src/polyfills.ts

```diff
@@ -50,7 +50,7 @@
 
 /** Evergreen browsers require these. **/
 // Used for reflect-metadata in JIT. If you use AOT (and only Angular decorators), you can remove.
-import 'core-js/es7/reflect';
+import 'core-js/es/reflect';
```

Comment:
Is this needed?

Reply:
This was actually broken before and was pretty much _accidentally_ working.  An old version of core-js was still sitting in the node modules due to the yarn lock file having a multitude of old package versions.